libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -3888,11 +3888,13 @@ string YulUtilFunctions::storageSetToZeroFunction(Type const& _type)
 		if (_type.isValueType())
 			return Whiskers(R"(
 				function <functionName>(slot, offset) {
-					<store>(slot, offset, <zeroValue>())
+					let <values> := <zeroValue>()
```

Comment:
Isn't function type essentially a bytes4 and hence a single slot?

Reply:
It's the address and selector. In two slots.

Reply:
Thank you. So the zeroing function that gets called is this one right?

https://github.com/ethereum/solidity/blob/58e09776470787e2f9e6052e0d7c29d8bcc1374c/libsolidity/codegen/YulUtilFunctions.cpp#L3815-L3824

Reply:
Yes.